src/py42/modules/archive.py

```diff
@@ -34,9 +34,9 @@ def stream_from_backup(
             encryption_key (str or None, optional): A custom encryption key for decrypting an archive's
                 file contents, necessary for restoring files. This is only relevant to users with custom
                 key archive security. Defaults to None.
-            ignore_size_calc (bool, optional): When True, ignores figuring our accurate file counts and
-                sizes. Recommended to set to True for large restores. File count and sizes are used
-                for indicating progress.
+            file_size_calc_timeout (int, optional): Set to limit the amount of seconds spent calculating
+                file sizes when crafting the request. Set to 0 to ignore file sizes altogether. Defaults
+                to None.
```

Comment:
Should clarify here that defaulting to None means it uses a timeout of 10 seconds. Also, would it make sense to have `None` be "no timeout" so it just runs until the job completes, however long that might take?

Reply:
Yes, I agree with both of that and will put that in.